### DISCUSSION TITLE
Install from source code

### DISCUSSION THREAD

**Discussion by m7mdhka** [Canner/WrenAI#650] [Q&A] [answered]
Hello,
I want to ask how can I install it from the source code without launcher?

**Comment by cyyeh** [comment, marked as answer]
@m7mdhka hi, thanks for reaching out!

actually underneath launcher is script to run docker containers, so you could run Wren AI yourself by entering docker-compose command. FYI, https://github.com/Canner/WrenAI/tree/main/docker
